tools/layer-zero-example/contracts/hts/IHederaTokenService.sol

```diff
@@ -0,0 +1,799 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.4.9 <0.9.0;
+pragma experimental ABIEncoderV2;
+
+interface IHederaTokenService {
```

Comment:
Lots of methods and potentially a few structs that might not be needed to define here.
Again you can link in comment to `IHederaTokenService` in the smart contracts repo.
Once we get the npm package going ti'll be easier.

Reply:
Done.